engine/src/main/java/org/terasology/logic/characters/KinematicCharacterMover.java

```diff
@@ -94,6 +94,40 @@ public KinematicCharacterMover(WorldProvider wp, PhysicsEngine physicsEngine) {
         physics = physicsEngine;
     }
 
+    /**
```

Comment:
moving this method does not have substantial benefit at this point (could apply it later, though)